src/Analyzers/Core/Analyzers/RemoveUnnecessaryImports/AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer.cs

```diff
@@ -31,7 +31,7 @@ internal abstract class AbstractRemoveUnnecessaryImportsDiagnosticAnalyzer
         // ruleset editor or solution explorer. Setting messageFormat to empty string ensures that we won't display
         // this diagnostic in the preview pane header.
         private static readonly DiagnosticDescriptor s_fixableIdDescriptor =
-            new DiagnosticDescriptor(DiagnosticFixableId,
+            new(DiagnosticFixableId,
                                      title: "", messageFormat: "", category: "",
                                      defaultSeverity: DiagnosticSeverity.Hidden,
                                      isEnabledByDefault: true,
```

Comment:
@sharwell this is a bit unfortunate.  thoughts on if we should tweak the fixer to detect and adjust cases like this?

Reply:
I always saw these lines as the same incorrect indentation before and after this change. We could update the fixer to handle this case, but it gets complex quickly. Whether or not edge cases are handled, the most important thing is making sure that any shift to the left doesn't shift any further than a nested indent relative to the start of the line containing `new`.